apps/api/src/auth/service/auth.service.ts

```diff
@@ -140,50 +140,6 @@ export class AuthService {
     }
   }
 
-  private async createUserIfNotExists(
```

Comment:
This one needs to stay. I will see what the error is. Once the e2e tests are up, I will be removing this.